### DISCUSSION TITLE
Is 2 way data binding supported?

### DISCUSSION THREAD

**Discussion by martinlombana** [actarian/rxcomp#6] [Q&A]
Hi! I see that initially you wrote that 2 way data binding is not supported but now that text is striked out. So I am unsure if it supports it or not.

Thanks in advance!

**Comment by actarian** [comment]
@martinlombana yes, two way data binding is not supported, those two features have been stricked out for this reason, but maybe it is not clear. In order to keep the lib performant all the state updates are done hierarchically through manual `pushChanges` method.  
[example](https://github.com/actarian/rxcomp/blob/master/web/client/js/app.component.ts)
